include/sys/dtb.h

```diff
@@ -3,6 +3,8 @@
 
 #include <sys/types.h>
 
+#define DTB_ROOT_NODE 0
```

Comment:
Do you need to expose this outside?

Reply:
FTTB, I don't. This may change when I will make PRs regarding RISC-V drivers, but for now I will move it to the source file.